source/class/qx/core/Object.js

```diff
@@ -155,16 +155,26 @@ qx.Class.define("qx.core.Object",
 
     /**
      * Call the same method of the super class.
+     * 
+     * Either the compiler translate all calls to this.base
+     * into mypkg.MyBaseClass.prototype.myMethod.call(this, 123);
+     * this method is still needed for use in compile.js or playground
+     * which are not precompiled
      *
      * @param args {IArguments} the arguments variable of the calling method
      * @param varargs {var?} variable number of arguments passed to the overwritten function
      * @return {var} the return value of the method of the base class.
      */
     base : function(args, varargs)
     {
+      let func = args.callee.base;
```

Comment:
I don't know if this is of any relevance from now on, but it doesn't make any difference here if to use `var` instead of `let` as the declaration is the first statement in the function.

Reply:
Changed